Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM node:20.2-alpine as builder
-RUN apk update && apk add curl yarn python3 build-base gcc wget git --no-cache
-RUN curl -sf https://gobinaries.com/tj/node-prune | sh
+FROM node:lts-bullseye-slim as builder
+RUN apt-get update && apt-get install -y build-essential ca-certificates apt-utils nodejs npm curl wget git yarn python3 dumb-init --no-install-recommends
+RUN curl -k -sf https://gobinaries.com/tj/node-prune | sh
 
 WORKDIR /usr/src/app
 
 ENV npm_config_target_arch=x64
 ENV npm_config_target_platform=linux
-ENV npm_config_target_libc=musl
+ENV npm_config_target_libc=glibc
+ENV NODE_ENV=production

```

Comment:
`NODE_ENV` is set in the webpack config.

Reply:
Difference in where it is built, and where it matters.